affirm/src/main/java/com/affirm/android/AffirmUtils.java

```diff
@@ -140,8 +140,9 @@ static SpannableString createSpannableForText(
             logoDrawable = resources.getDrawable(affirmLogoType.getDrawableRes(affirmColor));
         }
 
-        int color = affirmColor.getColorRes() != -1
-                ? resources.getColor(affirmColor.getColorRes()) : -1;
+        // The value 0 is an invalid identifier.
+        int color = affirmColor.getColorRes() != 0
+                ? resources.getColor(affirmColor.getColorRes()) : 0;
```

Comment:
If 0 is an invalid identifier, why are you using it? Use a default color?

Reply:
Fixed @kevindavmoore